molecule/notify/requirements.yml

```diff
@@ -0,0 +1,7 @@
+---
+# PostgreSQL
+- name: postgresql
+  src: ANXS.postgresql
```

Comment:
Noting for previous comment regarding use of the [geerlingguy postgresql role](https://github.com/geerlingguy/ansible-role-postgresql)

Reply:
For the test suite it still works...